pkg/driver/controller_test.go

```diff
@@ -289,6 +290,90 @@ func TestCreateVolume(t *testing.T) {
 				mockCtl.Finish()
 			},
 		},
+		{
+			name: "success: normal with deploymentType PERSISTENT_2 and metadataConfiguration",
```

Comment:
I notice you didn't add a test in controller_test for efaEnabled. Is that intentional?